torch/csrc/api/include/torch/nn/functional/activation.h

```diff
@@ -40,6 +40,20 @@ inline Tensor logsigmoid(const Tensor& input) {
   return torch::log_sigmoid(input);
 }
 
+inline Tensor softmax(const Tensor& input, const SoftmaxOptions& options,
+                      c10::optional<torch::Dtype> dtype = c10::nullopt) {
+  int dim = options.dim();
```

Comment:
We should change here accordingly:
```suggestion
  int64_t dim = options.dim();
```

Reply:
Thanks. I fixed it.